### ISSUE TITLE
Support for the '-d' flag to set the doc type

### ISSUE THREAD

**Issue by gdamore** [bytesparadise/libasciidoc#616]
At present we only have "article" and "manpage" document types.  (We might later add book.)

The default is manpage.

This can be set via attributes, but for compatibility with asciidoctor a -d command line switch might be nice.

**Comment by xcoulon** [comment]
> At present we only have "article" and "manpage" document types. (We might later add book.)
> 
> The default is manpage.
> 

no, the default is `article`: https://github.com/bytesparadise/libasciidoc/blob/master/pkg/renderer/sgml/renderer.go#L136

> This can be set via attributes, but for compatibility with asciidoctor a -d command line switch might be nice.

**Comment by gdamore** [comment]
Doh, that's what I mean to say -- the default is article.  Thanks.
